torch/csrc/jit/codegen/cuda/glfdc/expr.cpp

```diff
@@ -0,0 +1,24 @@
+#include <torch/csrc/jit/codegen/cuda/glfdc/expr.h>
+
+#include <c10/util/Exception.h>
+
+using namespace torch::jit::fuser::cuda::glfdc;
+
+void ExprDAG::markSExprReuse(SExprRef ref) {
+  TORCH_INTERNAL_ASSERT(ref.index() < reused_subexpressions_.size());
+
+  bool was_reused = reused_subexpressions_[ref.index()];
+  reuse_count_ += std::size_t(was_reused ^ true);
+
+  if (was_reused)
+    return;
+
+  reused_subexpressions_[ref.index()] = true;
+  auto e = this->fetch(ref);
+
+  if (!isValue(e.lhs_))
```

Comment:
Wouldn't it be safer to use `isSExpr` instead of `isValue` as `!isValue` does not mean `isSExpr`?